lib/client/indexer/user.ts

```diff
@@ -223,4 +224,42 @@ export class FutarchyIndexerUserClient implements FutarchyUserClient {
       userAcct: new PublicKey(u.user_acct)
     })).sort((a, b) => b.totalVolume - a.totalVolume);
   }
+
+  async fetchUserDeposits(userAcct: string): Promise<UserDeposit[]> {
+    const { user_deposits } = await this.graphqlClient.query({
+      user_deposits: {
+        __args: {
+          where: {
+            user_acct: {
+              _eq: userAcct
+            }
+          }
+        },
+        user_acct: true,
+        token_amount: true,
+        mint_acct: true,
+        tx_sig: true,
+        token: {
```

Comment:
@R-K-H are we cool with one join on this? Or should we let the UI piece this together? I think since deposits could be displayed across multiple DAOs maybe it would be a good idea to include this?

Reply:
I think it's fine for now, if we need to optimize we know where to start looking.